packages/client/src/components/app/forms/MultiFieldSelect.svelte

```diff
@@ -20,6 +20,7 @@
   let fieldSchema
 
   $: flatOptions = optionsSource == null || optionsSource === "schema"
+  $: expandedValue = expand(fieldState?.value)
```

Comment:
The best idea here would be to expand the `defaultValue` prop rather than the `value`, and then pass that in to the `Field` component, so that it gets registered with the form properly. We could just call it `expandedDefaultValue`.

Reply:
I have expanded both now -- just expanding defaultValue gave me the same error as before. Value is also expected to be enumerable, but unless we expand it, it is a string.